agent/utils.py

```diff
@@ -65,6 +67,7 @@ def initialize_msf_rpc() -> msfrpc.MsfRpcClient:
     """
     command = ["msfrpcd", "-P", MSFRPCD_PWD, "-p", "55555"]
     with subprocess.Popen(command):
+        time.sleep(60)
```

Comment:
move to a const, how did you come up with 60?

Reply:
> move to a const, how did you come up with 60?

just a buffer for the msfrpcd to initialize, locally it's instantaneous but on machines it seems to take couple seconds